rune/libenclave/internal/runtime/libenclave_v1.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+package runtime_proto;
+
+service Libenclave {
+	rpc GetVersion(google.protobuf.Empty) returns (GetVersionResponse);
+	rpc GetCapability(google.protobuf.Empty) returns (GetCapabilityResponse);
+	rpc Create(CreateRequest) returns (CreateResponse);
+	rpc Delete(DeleteRequest) returns (google.protobuf.Empty);
+	rpc Init(InitRequest) returns (InitResponse);
+	rpc Spawn(SpawnRequest) returns (SpawnResponse);
+	rpc Exec(ExecRequest) returns (ExecResponse);
+	rpc Kill(KillRequest) returns (KillResponse);
+	rpc Attest(AttestRequest) returns (AttestResponse);
+}
+
+message GetVersionResponse {
+	uint32 version = 1;
+}
+
+message GetCapabilityResponse {
+	string capability = 1;
+}
+
+message CreateRequest {
+	string loglevel = 1;
+	string bundle = 2;
+	string stdin = 3;
+	string stdout = 4;
+	string stderr = 5;
+	string cpus = 6;
+	string memory = 7;
+	string kernel = 8;
+	string vsock = 9;
```

Comment:
We need a more abstract description for the create request. For example, sgx enclave runtimes doesn't have the concept of kernel and vsock.